packages/api/src/graphql/article.ts

```diff
@@ -128,6 +128,20 @@ export const GraphQLArticleRevision = new GraphQLObjectType<ArticleRevision, Con
 
     canonicalUrl: {type: GraphQLString},
 
+    url: {
+      type: GraphQLNonNull(GraphQLString),
+      resolve: createProxyingResolver(({slug}, args, {urlAdapter}, info) => {
+        const id = info.variableValues.id
```

Comment:
I think it's possible to pass `id` through the `args`. This would be cleaner because it's not guaranteed that the `article` query always has the `id`.

Reply:
@roosnic1  Do you know how to pass smth in `args`? I am tried and couldn't get it working.

Reply:
@bishoymelek i checked again and realized that the args refer to the query args. So the `info.variableValues.id` is the right way. But i suggest you also check for `!id` on line 136.